.github/workflows/post-commit.yml

```diff
@@ -27,7 +27,7 @@ concurrency:
   cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 jobs:
   build-info:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
I don't think these are required changes for the issue.

Reply:
Not only is it not related to this change, it causes Findbugs errors.

https://github.com/Cyrill/ozone/runs/7952291029?check_suite_focus=true#step:6:8

@Cyrill can you please revert this part of the change?

Reply:
The verification fails with the following error without it:
```
[build-info](https://github.com/Cyrill/ozone/runs/7952516390?check_suite_focus=true)
This is a scheduled Ubuntu-18.04 brownout. The Ubuntu-18.04 environment is deprecated and will be removed on December 1st, 2022. For more details, see https://github.com/actions/runner-images/issues/6002
```

Reply:
Thanks @Cyrill for the info on the failure.  It seems this failure is only temporary, it will happen periodically until final removal of the 18.04 image, but in other periods builds will be successful.

This should be fixed as part of a separate Jira issue.  Would you like to file that and submit your fix?

Reply:
> @adoroszlai could you please approve build workflows?

@myskov Sorry, but it's [failing](https://github.com/Cyrill/ozone/runs/7952291029?check_suite_focus=true#step:6:8) in @Cyrill's fork (at findbugs), so this will need to wait.  I don't think it makes sense to start PR workflow until we get a clean run in fork.

Reply:
@adoroszlai Reverted. 
But I'm still puzzled with the following error. What is the best way to resolve it?
<img width="1377" alt="image" src="https://user-images.githubusercontent.com/56274/185942786-3dc93f12-bc1a-45cf-8749-5307de12b905.png">

Reply:
#3707 and #3708 (for HDDS-7159) are merged now.